mysql-test/suite/tianmu/t/auto_increment.test

```diff
@@ -0,0 +1,154 @@
+--source include/have_tianmu.inc
+#
+# Test auto_increment with TIANMU
+#
+
+# test auto increment
+--disable_warnings
+drop table if exists t_auto_inc;
```

Comment:
It's better to use your own database you created before the following tests start, and drop the database when test finished.

Reply:
ACK, I will modify it.

Reply:
Add the following content：
```
# Clean UP
DROP DATABASE auto_increment_db;
```